tests/NodeTest.php

```diff
@@ -224,7 +224,6 @@ public function testCategoryMovesUp()
     public function testFailsToInsertIntoChild()
     {
         $this->expectException(Exception::class);
-
```

Comment:
Similar comment as before. If possible I would like to not change the formatting. In this case, keep the blank line.

Reply:
Fixed.